src/main/java/com/snowflake/kafka/connector/internal/EnableLogging.java

```diff
@@ -0,0 +1,46 @@
+package com.snowflake.kafka.connector.internal;
```

Comment:
please add copyright

Reply:
I also noticed other files missing copyrights. Is the goal to have every file contain the copyright at the beginning?

Reply:
yes

Reply:
created corresponding jira